src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -224,8 +226,20 @@ class NotebookEditor extends Component {
         }
       }))
 
-      const { name: workspaceName } = this.props
-      this.setState({ url: `${clusterUrl}/notebooks/${workspaceName}/${notebookName}` })
+      const { name: workspaceName, app } = this.props
+      if (app === 'lab') {
+        this.setState({ url: `${clusterUrl}/${app}/tree/${workspaceName}/${notebookName}` })
+        pushNotification({
+          type: 'warning',
+          dismissable: { click: true },
+          dismiss: { duration: 30000 },
+          content: h(pushNotificationMessage, {
+            title: 'Autosave occurs every 2 minutes',
+            content: 'JupyterLab autosaves every 2 min. Please remember to save your notebook before exiting the window.'
+          }),
```

Comment:
I would add "by clicking the save icon"

Also:
"JupyterLab is new in Terra. We are working to improve its integration. Please contact us with any questions or feedback you may have."